src/main/java/dk/cngroup/lentils/service/convertors/CypherModelMapperWrapper.java

```diff
@@ -0,0 +1,72 @@
+package dk.cngroup.lentils.service.convertors;
+
+import dk.cngroup.lentils.dto.CypherFormDTO;
+import dk.cngroup.lentils.entity.Cypher;
+import dk.cngroup.lentils.entity.User;
+import dk.cngroup.lentils.service.UserService;
+import org.modelmapper.PropertyMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Component;
+
+import java.util.stream.Collectors;
+
+@Component
+@Primary
+public class CypherModelMapperWrapper extends ModelMapperWrapper {
```

Comment:
Za mě bych to možná nepojmenovával až takhle. Ale třeba jen CypherMapper.
Když se změní vnitřně ta implementace, místo modelMapperu se dá něco jiného nebo něco takového, nebude to tolik vadit.